src/java/com/google/cloud/bigquery/dwhassessment/hooks/logger/EventRecordConstructor.java

```diff
@@ -286,7 +303,7 @@ private static ExecutionMode getExecutionMode(QueryPlan plan) {
     return ExecutionMode.NONE;
   }
 
-  private static String getQueueName(ExecutionMode mode, HiveConf conf) {
+  private static String getSupposedQueueName(ExecutionMode mode, HiveConf conf) {
```

Comment:
Are `retrieveRealQueueName` and `getSupposedQueueName` methods related? If so, I would suggest to:
- rename both to start with `get` or `retrieve`
- make the order of parameters consistent.

Reply:
Done